api/src/data_migration/transformation/transform_util.py

```diff
@@ -257,10 +254,6 @@ def transform_opportunity_summary(
     target_summary.agency_email_address = source_summary.ac_email_addr
     target_summary.agency_email_address_description = source_summary.ac_email_desc
     target_summary.can_send_mail = convert_yn_bool(source_summary.sendmail)
-    target_summary.publisher_profile_id = source_summary.publisher_profile_id
-    target_summary.publisher_user_id = source_summary.publisheruid
-    target_summary.updated_by = source_summary.last_upd_id
```

Comment:
Some of the test failures are because there are a few more fields that need to be deleted in this file. For opportunity attachment, applicant type, funding instrument, and funding category, they each have a function for transforming that also set values.